modules/runtime/pages/process-manager-application.adoc

```diff
@@ -0,0 +1,59 @@
+= Creating a process manager application
+:description: This page explains how to create your own custom process manager application
+
+[NOTE]
+====
+For Enterprise, Performance, Efficiency and Teamwork editions only.
+====
+
+This page explains how to create your own custom process manager application. +
+This application functions like the Administrator application, but has a smaller scope, one that only controls processes, cases and tasks. +
+Creating such an application follows the same steps as the creation of any xref:applications:application-creation.adoc[other custom application].
+
+== Pages
+
+The main menu of the application will require four pages.
+
+* Monitoring
+* Process list
+* Case list
+* Task list
+
+In addition to those, there are some orphan pages that are required by the above ones :
+
+* Process details
+* Process visualization
+* Case details
+* Case visualization
+* Task details
+
+== Page modifications
+
+The above-mentioned menu pages are the same as the ones that are part of the Administrator application, but they only show the processes/cases/tasks that the user supervises. +
+In order to make the menu pages do so, you need to export them from the Administrator Resource List and edit them using the UI Designer.
+You don't need to export and modify the orphan pages.
+
+[WARNING]
+====
+
+Currently, the Monitoring page is made using a different technology than UI Designer, and thus it cannot be edited.
+====
+
+The modifications that you will need to do are quite simple:
```

Comment:
I do not recommend to qualify the edition of a complex page with the UID as something quite simple, it is probably the case for you, one of the dev of the original page, but i'm pretty sure that many users will find this complicated. 
As always, i suggest to avoid adjectives like 'easy, simple...', you do not know the level of your reader.

An other point is that i suggest you to be a bit more explicit about which pages have to be modified, I had to read the text 3 times to get it.

```suggestion
Perform the following modifications to the 4 menu pages:
```